src/main/java/bio/terra/workspace/service/workspace/WorkspaceService.java

```diff
@@ -48,29 +50,37 @@ public WorkspaceService(
 
   /** Create a workspace with the specified parameters. Returns workspaceID of the new workspace. */
   @Traced
-  public UUID createWorkspace(
-      UUID workspaceId,
-      Optional<SpendProfileId> spendProfileId,
-      WorkspaceStage workspaceStage,
-      AuthenticatedUserRequest userReq) {
+  public UUID createWorkspace(WorkspaceRequest internalRequest, AuthenticatedUserRequest userReq) {
```

Comment:
not wild about `internalRequest` variable name. What's wrong with `workspaceRequest` or `request`? At this layer, how is this more `internal` than anything else?

Reply:
`internalRequest` makes sense to me at the controller level and I plumbed the same name through here. `workspaceRequest` sgtm